.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+e2eTest: &e2eTest
+  machine: true
+  working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/kvm-operator
```

Comment:
```suggestion
  working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/bridge-operator
```

Reply:
I copied the wrong file. Is simpler now.